File: bootcamp/week-04-primeros_tests_automatizados_con_python/2-practicas/ejercicio-02-primeros-tests-python/starter/src/user_utils.py
def is_adult(age: int) -> bool:
    return age >= 18


def calculate_discount(price: float, percent: int) -> float:
    if percent < 0 or percent > 100:
        raise ValueError("Invalid percent")
    return price - (price * percent) / 100


def is_valid_email(email: str) -> bool:
    return "@" in email and "." in email

File: bootcamp/week-03-primeros_tests_automatizados_con_javascript/2-practicas/ejercicio-02-primeros-tests/solution/src/user-utils.js
function isAdult(age) {
  return age >= 18;
}

function calculateDiscount(price, percent) {
  if (percent < 0 || percent > 100) {
    throw new Error("Invalid percent");
  }
  return price - (price * percent) / 100;
}

function isValidEmail(email) {
  return email.includes("@") && email.includes(".");
}

module.exports = { isAdult, calculateDiscount, isValidEmail };
